src/Sulu/Bundle/ContentBundle/Resources/phpcr-migrations/Version201607181533.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContentBundle;
+
+use PHPCR\ImportUUIDBehaviorInterface;
+use PHPCR\Migrations\VersionInterface;
+use PHPCR\SessionInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+class Version201607181533 implements VersionInterface, ContainerAwareInterface
+{
+    /**
+     * @var ContainerInterface
+     */
+    private $container;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function up(SessionInterface $session)
+    {
+        $defaultSession = $this->container->get('sulu_document_manager.default_session');
+        $liveSession = $this->container->get('sulu_document_manager.live_session');
+        $session->getWorkspace()->createWorkspace($liveSession->getWorkspace()->getName());
+        $queryManager = $liveSession->getWorkspace()->getQueryManager();
+
+        $fileName = tempnam(sys_get_temp_dir(), 'sulu-publishing');
+        $file = fopen($fileName, 'w');
+        $defaultSession->exportSystemView('/cmf', $file, false, false);
```

Comment:
@chirimoya Do we agree that copying the `cmf` node is enough? Then we could say we only have the `jcr:versions` node on the draft workspace, and future migrations have to adapt both workspaces.